db/migrate/20241010194153_destroy_route_eleven_matches.rb

```diff
@@ -0,0 +1,5 @@
+class DestroyRouteElevenMatches < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
```suggestion
  def up
```

Reply:
This has been added in the latest commit